### DISCUSSION TITLE
FlightDataRecorder

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#6389] [Q&A] [answered]
hello,

I know there is a FlightDataRecorder feature on the A32NX. 
it creates a folder in 
C:\Users\andre\AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages\flybywire-aircraft-a320-neo\

the WORK folder contains a lot big files - the fdr files. BUT i turned this features off in the FlightDataRecoder.ini

any Ideas where the Files come from? can I delete it?
![FDR](https://user-images.githubusercontent.com/72918017/146695367-af4a4d26-9101-4313-886c-792382885c8b.jpg)

**Comment by aguther** [comment, marked as answer]
The file format changed. Delete the ini file load the plane which will create a new ini file with defaults. then you can switch the enabled back to false and it should work.

I would suggest to limit the number of files to 2 or 3 instead of completely disabling it. That way you can still send us the file if you need support, it contains a lot of parameters that can help in that.

You can also have a look yourself into the files:
https://github.com/flybywiresim/fdr-analysis
